### DISCUSSION TITLE
Welcome to awesome Discussions!

### DISCUSSION THREAD

**Discussion by actionless** [awesomeWM/awesome#3812] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Aire-One** [comment]
Finally! 🎉

**Comment by Aneeqasif** [comment]
will awesome support wayland?

**Reply by sclu1034** [reply]
We lack both time and expertise to realize such a project. So it's unlikely that we will see Wayland support in the near future.

**Reply by unai-ndz** [reply]
WayCooler tried to do this, sadly it was abandoned but the author explains some things here. It's long so if you wanna skip some stuff it talks about awesome at the end of the post.

[WayCooler post-mortem](http://way-cooler.org/blog/2020/01/09/way-cooler-post-mortem.html)

**Reply by sclu1034** [reply]
@foxjaw I guess whatever you read was an inaccurate re-hash of the official info. The only thing that Fedora 40 will drop is the X11 **Plasma Session**.
Xorg packages and friends are deprecated, but still pending "to be removed in a future major RHEL release" ([fedoraproject.org](https://fedoraproject.org/wiki/Changes/KDE_Plasma_6#Why_drop_the_X11_session?)).

So despite being about a version from 1.5 years ago, Elv's comment still stands: Xorg won't be running by default, but you should still be able to install it as usual.

**Reply by sclu1034** [reply]
Feel free to get started then.
None of us ever said that we wouldn't want Wayland compatibility. But we all have our own lives to live, and our own things to do.
People can pout all they want, this is still a hobby project of people donating their free time. Usually based on their own needs first.

But plenty of bug bounty sites exist if you feel like you need to raise the incentive.
